sljit_src/sljitWXExecAllocator.c

```diff
@@ -61,8 +70,9 @@
 SLJIT_API_FUNC_ATTRIBUTE void* sljit_malloc_exec(sljit_uw size)
 {
 	sljit_uw* ptr;
+	sljit_uw page_mask = (sljit_uw)get_page_alignment();
 
-	size += sizeof(sljit_uw);
+	size = (size + sizeof(sljit_uw) + page_mask) & ~page_mask;
```

Comment:
@krytarowski do you have any comments on this and if we could ifdef it out so it only applies to the NetBSD kernel?

Reply:
I'm waiting for @alnsn to update sljit for the NetBSD kernel.